content/applications/general/calendars/google/google_calendar_credentials.rst

```diff
@@ -2,89 +2,130 @@
 Synchronize Google Calendar with Odoo
 =====================================
 
-Odoo is perfectly integrated with Google Calendar so that you 
-can see & manage your meetings from both platforms 
-(updates go through both directions).
+Synchronize Google Calendar with Odoo to see and manage meetings from both platforms (updates go
+in both directions). This integration helps organize your schedule so you never miss a meeting.
 
 Setup in Google
 ===============
-- Go to `Google APIs platform <https://console.developers.google.com>`__ 
-  to generate Google Calendar API credentials. Log in with your Google account. 
 
-- Go to the API & Services page.
+Enable Google Calendar API
+--------------------------
 
-.. image:: media/google_calendar_credentials00.png
-    :align: center
+To begin, you need to create a new Google API project and enable the Google Calendar API. First, go
+to the `Google API Console <https://console.developers.google.com_>`_ and log into your Google
+account.
 
-- Search for *Google Calendar API* and select it.
+.. note::
+   If this is your first time visiting this page, Google will prompt you to enter your country and
```

Comment:
your --> a given that the requirement is to enter _a_ country (not necessarily _yours_)

```suggestion
   If this is your first time visiting this page, Google will prompt you to enter a country and
```